src/app/helptext/sharing/nfs/nfs.ts

```diff
@@ -90,7 +90,11 @@ export const helptext_sharing_nfs = {
 
   dialog_enable_service_title: T("Enable service"),
   dialog_enable_service_message: T("Enable this service?"),
-  dialog_enable_service_button: T("Enable Service"),
+  dialog_enable_service_button: T("Enable Service"),    
+  
+  dialog_delete_title: T("Delete"),
+  dialog_delete_message: T("Stop sharing NFS? The sharing configuration will be deleted\
+(but the underlying dataset will not)."),
```

Comment:
As above.  Also, can't we factor out this dialog and pass in the message as a parameter?  It seems like a lot of work to maintain a different one for each sharing system.